flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -91,6 +91,12 @@ private FlinkConfigOptions() {}
           .defaultValue(false)
           .withDescription("Use the FLIP-27 based Iceberg source implementation.");
 
+  public static final ConfigOption<Boolean> TABLE_EXEC_ICEBERG_USE_FLIP143_SINK =
```

Comment:
I think this config name is not very good. We should not include "flip" in it. 
How about calling `use-v2-sink`?